code/modules/research/designs/medical.dm

```diff
@@ -139,3 +139,14 @@
 	materials = list (MAT_IRON = 1000, MAT_GLASS = 1000)
 	category = "Medical"
 	build_path = /obj/item/device/antibody_scanner
+	
+	/datum/design/switchtool
+	name = "Surgeon's Switchtool"
+	desc = "A switchtool containing most of the necessary items for impromptu surgery. For the surgeon on the go."
+	id = "switchtool"
+	req_tech = list("materials" = 5, "bluespace" = 3, "biotech" = 3)
+	build_type = PROTOLATHE
+	materials = list (MAT_IRON = 10000, MAT_GLASS = 5000)
+	category = "Medical"
+	build_path = /obj/item/weapon/switchtool/surgery
```

Comment:
DM doesn't give the slightest of a shit about double var overrides, it uses the ones compiled last.